core/commands/root.go

```diff
@@ -75,6 +75,9 @@ TOOL COMMANDS
   commands      List all available commands
   log           Manage and show logs of running daemon
 
+UTILITIES
+  multibase     Encode and decode files with multibase format
```

Comment:
Could we move it up and include existing `cid` in this new section? 
It will make it easier for people to discover them and their purpose.

```
    block         Interact with raw blocks in the datastore

  ENCODING COMMANDS
    cid           Convert and discover properties of CIDs
    multibase     Encode and decode files and stdin with multibase format

  ADVANCED COMMANDS
    daemon        Start a long-running daemon process
````